common/transport/http/src/rest_api_client.ts

```diff
@@ -36,7 +36,7 @@ export interface HttpTransportError extends Error {
  * @throws {ArgumentError}   If the config argument is missing a host or sharedAccessSignature error
  */
 export class RestApiClient {
-  private _iotHubPublicScope: string[] = ['https://iothubs.azure.net/.default'];
+  private _iotHubPublicScope: string = 'https://iothubs.azure.net/.default';
```

Comment:
does this affect the getToken credential call signature, since you're removing the array wrap?

Reply:
It accepts a string or an array of strings. In the AMQP client, we were using a string, so I changed it here to be consistent.